lib/game_team.rb

```diff
@@ -0,0 +1,16 @@
+class GameTeam
+  attr_reader :game_id, :team_id, :HoA, :result, :settled_in, :head_coach,
+  :goals, :shots, :tackles
+
```

Comment:
This looks great, very similar to the team which I think is what we want, so looks good!